### PR TITLE
System Check Script #6: Radio 

### PR DESCRIPTION
**Pull request by yashika-batra**
Radio Test: clean up + style and documentation fixes

### CODE REVIEW

File: state_machine/applications/system-check/tests/radio_test.py [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+async def radio_test(test_type):
+    """
+    Run radio tests given an input - Send, Receive, or Both
+    Return if there was at least one success, as well as the
+    # of transmission/reception attempts and successes
+    """
+
+    attempts = 5
+
+    if test_type == "Both":
```
Comment (thetazero) [inline state_machine/applications/system-check/tests/radio_test.py:26]: split these test types into separate functions please

File: state_machine/applications/system-check/tests/radio_test.py [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+        success = False
+        success_count = 0
+
+        print("Transmission and Reception Test: " +
+              "Sending Message and Awaiting Groundstation Response (25s)")
+
+        for i in range(attempts):
+            # send beacon text and keep listening
+            cubesat.radio.send(send_text, keep_listening=True)
```
Comment (thetazero) [inline state_machine/applications/system-check/tests/radio_test.py:35]: pretty sure you can just do send_with_ack

File: state_machine/applications/system-check/tests/radio_test.py [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+            print("Radio Test complete.")
+            print("")
+            return result_dict
+
+        # if the test fails
+        else:
+            # individually test reception and update result dict
+            receive_result, receive_count, receive_attempts = radio_test(
+                "Receive")
+            receive_val_string = ("Succeeded in receiving message" +
```
Comment (thetazero) [inline state_machine/applications/system-check/tests/radio_test.py:167]: f string

**Review by thetazero — CHANGES_REQUESTED** [review]
I'm not sure if we want to use this code, I've wrote a test script that does something a little more advanced for the radio, and is compatible with my new updates as well.

We may want to hold off on this one for now, and have me port my work into the check script.

**Comment by yashika-batra** [comment]
Leaving this as is for now, since we'll most likely be using Losha's radio test as part of the check script.

**Pull request closed without merging**